examples/meta/src/distance/manhattan_distance.sg

```diff
@@ -0,0 +1,18 @@
+CSVFile f_feats_a("../../data/fm_train_real.dat")
+CSVFile f_feats_b("../../data/fm_test_real.dat")
+
+#![create_features]
+RealFeatures features_a(f_feats_a)
+RealFeatures features_b(f_feats_b)
+#![create_features]
+
+#![create_distance]
+ManhattanMetric distance(features_a, features_b)
+#![create_distance]
+
+#![train_and_init]
+RealMatrix dm_train = distance.get_distance_matrix()
+distance.init(features_a, features_b)
```

Comment:
this is the same as above, so both of your distance matrices will be the same.
Actually, why don't we just remove everything after this line, and only compute the distance_ab.
It is pretty obvious how to do other combinations.....